jobs/resource_job.go

```diff
@@ -406,10 +432,20 @@ func jobSettingsSchema(s *map[string]*schema.Schema, prefix string) {
 	}
 }
 
+func gitSourceSchema(s *schema.Resource, prefix string) {
```

Comment:
one-letter variable names should be the first letters of types they refer to - at least that's how most of people read it in the middle of the night ;) so `r *schema.Resource`. Otherwise a longer name :)